output.go

```diff
@@ -97,7 +97,7 @@ type VMOutput struct {
 	// This data tells the network how to update the account data.
 	// It can contain new accounts or existing changed accounts.
 	// Note: the current implementation might also retrieve accounts that were not changed.
```

Comment:
I'm guessing here the map key is the account address, right?

Reply:
Indeed. The comment is now up to date.